README.md

```diff
@@ -1,2 +1,130 @@
+| playbook_path  | string        | ci/ansible/playbook.yaml | The path to the file relative to the repository from which the deployment is made, used in Ansistrano deploy task |
+| deploy_path    | string        | ~/deploy/                | The path to deploy on the remote server                                                                           |
+| deploy_repo    | string        | git@github.com:saritasa-nest/ceai-wordpress.git | The path to deploy on the remote server                                                    |
+| deploy_branch  | string        | develop                  | The branch from which the deployment is made                                                                      |
+| python_version | number        | 3.11.4 | Each python version supports a certain range of ansible versions, so we need to specify the current version of python in order to install the latest version of ansible. |
+| runner         | string        | ubuntu-latest | The runner on which workflow is running: ubuntu, macos, windows, etc                                                         |
+
+### Inputs secrets
+
+| Variable       | Discription                                                    |
+| ---------------|:---------------------------------------------------------------|
+| ssh_private_key | The private key for connect to production server via ssh      |
+
+### Examples
+
+#### Workflow for deploy
+
+This implementation is used in the [CEAI-Wordpress project](https://github.com/saritasa-nest/ceai-wordpress/tree/main/.github/workflows) project to deploy Wordpress on shared hosting.
+
+```bash
+name: Deploy to Bluehost
+on:
+  workflow_dispatch:
+
+jobs:
+  deploy:
+    uses: saritasa-nest/saritasa-github-actions/.github/workflows/ci-wordpress.yaml@v2.4-dev.1
+    with:
+      ENVIRONMENT: production                                      # The Environment for deploy
+      SERVER_HOST: 50.87.253.20                                    # The pruduction server
+      SSH_DEPLOY_USER: ${{ vars.SSH_DEPLOY_USER }}                 # The username for connect to production server via ssh
+      PLAYBOOK_PATH: ci/ansible/ansistrano-deploy.yml              # The path to the file relative to the repository from which the deployment is made
+      DEPLOY_PATH: ~/deploy/                                       # The path to deploy on the production server
+      DEPLOY_REPO: git@github.com:saritasa-nest/ceai-wordpress.git # The repository for deploy
+      DEPLOY_BRANCH: develop                                       # The branche for deploy
+      PYTHON_VERSION: 3.11                                         # The version of python
+      RUNNER: ubuntu-latest                                        # The type of github runner
+
+    secrets:
+      ssh_private_key: ${{ secrets.SSH_PRIVATE_KEY }}
+
+```
+
+### Ansistrano deploy playbook
+
+```bash
+---
```

Comment:
if you reference `PLAYBOOK_PATH: ci/ansible/ansistrano-deploy.yml` then name this file accordingly vs just YAML snapshot.

Reply:
I changed name of playbook in example

Reply:
I Added filenames for examples section